nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/configuration/local.md

```diff
@@ -25,18 +25,18 @@ The public docker images are tagged with the same version numbers as the code, w
 ## Singularity image
 Many HPC environments are not able to run Docker due to security issues. [Singularity](http://singularity.lbl.gov/) is a tool designed to run on such HPC systems which is very similar to Docker. Even better, it can use create images directly from dockerhub.
 
-To use the singularity image for a single run, use `-with-singularity 'docker://{{ cookiecutter.dockerhub_slug }}'`. This will download the docker container from dockerhub and create a singularity image for you dynamically.
+To use the singularity image for a single run, use `-profile singularity 'shub://nfcore/tools:[VERSION]'`. This will download the docker container from dockerhub and create a singularity image for you dynamically.
```

Comment:
Shouldn't it be shub://nfcore/{{ cookiecutter.pipeline_slug }} like after ?

Reply:
Maybe......? ;-)

Reply:
Yes it should :-)

Reply:
+ we're `shub://nf-core/tools:[VERSION]` on singularity hub.

Reply:
No, that would be for `-with-singularity`. `-profile singularity` is already set up with the shub address so needs no further arguments.

I updated this before merging.

Reply:
Oh perfect nice :-)